contrib/k8s/examples/mysql/mysqldb-database-only-binding.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: servicecatalog.k8s.io/v1beta1
+kind: ServiceBinding
+metadata:
+  name: mysql-database-only-binding
```

Comment:
Since we can only bind to databases anyway (can't bind to dbms), do we really need to qualify this with "database-only"?

In fact... as I think about it more...

Can't we have one binding example that works with both the db that's provisioned as part of all-in-one _and_ a db that is provisioned as a child of a dbms?

Reply:
We can only do that if the instance name is the same. Should we make the instance name the same and assume they cannot be provisioned at the same time? I'm fine either way (this applies across all three)

Reply:
> We can only do that if the instance name is the same.

I thought I was ok with that, but then...

> they cannot be provisioned at the same time?

That will screw over _someone_ who's just kicking the tires.

Maybe we can can just drop "only" from the binding name?